Signal/src/Models/ThreadModel.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+import Foundation
+
+@objc
+public class ThreadModel: NSObject {
```

Comment:
Shouldn't we call this `ThreadViewModel`?

Reply:
sure, done!